views/signup.ejs

```diff
@@ -0,0 +1,53 @@
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/open-fonts@1.1.1/fonts/inter.min.css">
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@exampledev/new.css@1.1.2/new.min.css">
+
+<h1>Learn Databases</h1>
+
+<p>Create Account</p>
+
+<div>
+  <input
+    class="square email"
+    type="text"
+    placeholder="Email Address"
+    required
+    />
+</div>
+
+<div>
+  <input
+    class="square username"
+    type="text"
+    placeholder="Username"
+    required
+    />
+</div>
+
+<div>
+  <button class="square submit">Create Account</button>
+</div>
+
+<div>
+  Already have an account? <a href="/signin">Login</a>
+</div>
+
+<script>
+  const submit = document.querySelector('.submit');
+  const check = document.querySelector('.check');
+  const username = document.querySelector('.username')
+  const email = document.querySelector('.email')
+  submit.addEventListener('click', () => {
+    fetch('/api/users', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        username: username.value,
+        email: email.value
+      })
+    }).then(r => r.json()).then((r) => {
```

Comment:
（r）can just be written as () since r is not used...

Reply:
I'm going to send an email with the response

Reply:
@greenc123 this actually redirects. This code here is the front end.